### DISCUSSION TITLE
Continue with Google Button Missing

### DISCUSSION THREAD

**Discussion by txprojects88** [plausible/analytics#1561] [Self-Hosted Support]
I have set both the GOOGLE_CLIENT_ID and GOOGLE_CLIENT_SECRET and restarted docker but the Continue with Google button is still missing. Any ideas? Thanks!

<br class="Apple-interchange-newline">

<img width="812" alt="CleanShot 2021-12-22 at 17 14 24@2x" src="https://user-images.githubusercontent.com/81271945/147161130-579b8e1b-d7a1-4381-b54c-d2f237e5ba91.png">

**Comment by gambolputty** [comment]
Same here :/

**Comment by ukutaht** [comment]
I've just tested and it works OK. How did you restart your docker container? It's possible the configuration change didn't take effect

**Reply by gambolputty** [reply]
I followed the instructions [in the docs](https://plausible.io/docs/self-hosting-configuration#task-one-create-an-oauth-client) and added the client id and client secret to the `plausible-conf.env` and restarted the containers with `docker-compose restart`.

**Reply by alessio-zamparelli** [reply]
same error, same attempt

**Reply by ukutaht** [reply]
Looks like the instructions are wrong. `docker-compose restart` is not the right command. From docker's [documentantion](https://docs.docker.com/compose/reference/restart/)

> For example, changes to environment variables (which are added after a container is built, but before the container’s command is executed) are not updated after restarting.

In order to have the environment variable change reflected in your installation, try doing:
```bash
$ docker-compose down --remove-orphans
$ docker-compose up -d
```

I'll fix our docs

**Reply by gambolputty** [reply]
Thank you for the clarification! It works now.
